packages/backend/src/api/controllers/report/generateMain.ts

```diff
@@ -11,23 +10,17 @@ export function generateMain(
   sixHourlyReports: AggregateReportRecord[],
   dailyReports: AggregateReportRecord[],
   latestReports: ReportRecord[],
-  projects: ReportProject[],
+  projectIds: ProjectId[],
 ): ApiMain {
+  const reports = [hourlyReports, sixHourlyReports, dailyReports] as const
   return {
-    charts: getProjectCharts(
-      hourlyReports,
-      sixHourlyReports,
-      dailyReports,
-      ProjectId.ALL,
-    ),
-    projects: projects.reduce<ApiMain['projects']>((acc, { projectId }) => {
+    charts: getProjectCharts(...reports, ProjectId.LAYER2S),
+    layers2s: getProjectCharts(...reports, ProjectId.LAYER2S),
```

Comment:
Are `charts` and `layer2s` keys supposed to have the same values?

Reply:
That's exactly right. In the near future the `charts` property is going to get removed, but it needs to stay here as long as the frontend depends on it.